nuxeo-drive-core/src/main/java/org/nuxeo/drive/listener/NuxeoDriveSyncRootCopyListener.java

```diff
@@ -54,6 +54,8 @@ public void handleEvent(Event event) {
         DocumentModelList syncRoots = getSyncRoots(doc, session);
         for (DocumentModel syncRoot : syncRoots) {
             syncRoot.setPropertyValue(NuxeoDriveManagerImpl.DRIVE_SUBSCRIPTIONS_PROPERTY, null);
+            syncRoot.putContextData("source", "drive");
+            syncRoot.putContextData(CoreSession.SOURCE, "drive");
```

Comment:
Duplicate? Alos, is that needed as the modification comes from the Drive server addon?